docs_src/splitting_apps/version.py

```diff
@@ -0,0 +1,8 @@
+import typer
+
+app = typer.Typer()
+
+
+@app.command()
+def version():
+    typer.echo("My CLI Version 1.0")
```

Comment:
```suggestion
    print("My CLI Version 1.0")
```

Lately I've been trying to switch to `print()`, as `echo()` doesn't do that much on top of `print()`. And for richer format and colors, Rich is better. :sweat_smile: